projects/plugins/social/src/js/components/admin-page/header/index.jsx

```diff
@@ -7,14 +7,15 @@ import styles from './styles.module.scss';
 
 const AdminPageHeader = () => {
 	const activateLicenseUrl = getMyJetpackUrl( '#/add-license' );
+	const isWoaSite = Boolean( window.JetpackScriptData?.social?.is_woa_site );
 
 	return (
 		<div className={ styles.header }>
 			<span className={ styles.logo }>
 				<Logo />
 			</span>
 
-			{ ! hasSocialPaidFeatures() && (
+			{ ! hasSocialPaidFeatures() && ! isWoaSite && (
```

Comment:
Then here we can use `! getScriptData().site.host !== 'woa'` after

`import { getScriptData } from '@automattic/jetpack-script-data';`